pkg/cmd/dedicated/register/registercluster.go

```diff
@@ -0,0 +1,433 @@
+package register
```

Comment:
Can we look if this file can be split into multiple?

Reply:
this file should def be split up, generic cluster mgmt api logic etc needs to move into its own dir. 

I was planning on doing this after delivering the MVP